### PR TITLE
Review `check_qtr_format` function 

### PR DESCRIPTION
**Pull request by Jennit07**
Opening a draft PR for now. I think we need to review the care home methodology before deciding if we need the quarter calculation functions. I will highlight areas in the code which i am not sure about as i want to make sure we pick up the correct logic. 

For now i have concluded that we can remove the `check_qtr_format` function so have removed this from the code.

Closes Issue #886

### CODE REVIEW

File: R/calculate_stay.R [hunk 1]
```diff
@@ -51,6 +51,7 @@ calculate_stay <- function(year, start_date, end_date, sc_qtr = NULL) {
       lubridate::period(1L, "days")
     )
 
+    # check logic here for care home methodology
     dummy_end_date <- dplyr::case_when(
```
Comment (Jennit07) [inline R/calculate_stay.R:55]: Check care home logic here and decide if we need the quarter calculation functions

File: R/create_individual_file.R [hunk 2]
```diff
@@ -483,8 +483,10 @@ add_ch_columns <- function(episode_file, prefix, condition) {
       ch_ep_end = dplyr::if_else(
         eval(condition),
         .data$record_keydate2,
-        lubridate::NA_Date_  ),
-      # If end date is missing use the first day of next FY quarter
+        lubridate::NA_Date_
+      ),
+      # check logic here for care home methodology
+      # If end date is missing use the end of the FY quarter
```
Comment (Jennit07) [inline R/create_individual_file.R:489]: Check care home logic here and decide if we need the quarter calculation functions

**Comment by Jennit07** [comment]
Quarter calculation functions we need to review if they are needed. Currently used in care home methodology used to calculate stay. 

Script: get_fy_quarter_dates.R
- start_fy_quarter
- end_fy_quarter
- start_next_fy_quarter
- end_next_fy_quarter

**Comment by Jennit07** [comment]
Ready for review/merge

**Review by rchlv — APPROVED** [review]
Thanks for working on this Jen. This looks good to me, I'm happy to merge.

**Pull request merged**